Loop/Models/ServiceAuthentication/ShareService.swift

```diff
@@ -36,14 +38,23 @@ class ShareService: ServiceAuthentication {
                 value: url?.absoluteString,
                 options: [
                     (title: NSLocalizedString("US", comment: "U.S. share server option title"),
-                     value: DexcomShareURL.absoluteString)
+                     value: DexcomShareURL.absoluteString),
+                    (title: NSLocalizedString("Outside US", comment: "Outside US share server option title"),
+                     value: KnownShareServers.NON_US.rawValue)
+
                 ]
             )
         ]
 
+        if customServer.characters.count > 0 {
+            credentials[2].options?.append(
+                (title: NSLocalizedString("Custom", comment: "Custom share server option title"),
+                value: customServer))
+        }
+
         if let username = username, let password = password, url != nil {
             isAuthorized = true
-            client = ShareClient(username: username, password: password)
+            client = ShareClient(username: username, password: password, shareServer: url!.absoluteString)
```

Comment:
Yeah, agreed. Can you explain why the url != nil was added in the first place (rather than let url = url)..?